crates/tess2/src/tessellator.rs

```diff
@@ -11,6 +11,18 @@ use std::ptr;
 use std::slice;
 use tess2_sys::*;
 
+/// A unit struct that represents any error occuring during tesselation.
+#[derive(Debug)]
+pub struct TesselationError;
```

Comment:
In principle I think this is a good idea but I want to delay API breaking changes until a while later, So it would be great to remove them or put them out in another PR to avoid delaying everything else.

Reply:
I've made PR #665 with this change, and removed it from the current one.